doc/src/suite-config.rst

```diff
@@ -2052,10 +2052,47 @@ namespace (as for single-inheritance) is used for suite visualization
 purposes.
 
 
-Namespace Names
-^^^^^^^^^^^^^^^
+.. _TaskNames:
+
+Task and Namespace Names
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+There are restrictions on names that can be used for tasks and namespaces
+to ensure all tasks are processed and implemented correctly. Valid names
+must:
+
+1. *begin with* (or consist only of, as single-character task names are
+   allowed) either:
+
+   - an alphanumeric character, i.e. a letter in either upper or lower case
+     (``a-z`` or ``A-Z``), or a digit (``0-9``);
```

Comment:
Note: In Python 3, this will become any Unicode character that matches `\w` in a locale.